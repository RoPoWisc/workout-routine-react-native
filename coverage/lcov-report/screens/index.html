
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for screens</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../index.html">All files</a> screens</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">36.15% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>107/296</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">3.28% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>2/61</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">24.32% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>27/111</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">36.77% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>107/291</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line low'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="AddWorkout.js"><a href="AddWorkout.js.html">AddWorkout.js</a></td>
	<td data-value="66.67" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 66%"></div><div class="cover-empty" style="width: 34%"></div></div>
	</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="9" class="abs medium">6/9</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="9" class="abs medium">6/9</td>
	</tr>

<tr>
	<td class="file medium" data-value="EditWorkout.js"><a href="EditWorkout.js.html">EditWorkout.js</a></td>
	<td data-value="55.56" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 55%"></div><div class="cover-empty" style="width: 45%"></div></div>
	</td>
	<td data-value="55.56" class="pct medium">55.56%</td>
	<td data-value="9" class="abs medium">5/9</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="20" class="pct low">20%</td>
	<td data-value="5" class="abs low">1/5</td>
	<td data-value="55.56" class="pct medium">55.56%</td>
	<td data-value="9" class="abs medium">5/9</td>
	</tr>

<tr>
	<td class="file low" data-value="ForgotPassword.js"><a href="ForgotPassword.js.html">ForgotPassword.js</a></td>
	<td data-value="42.11" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 42%"></div><div class="cover-empty" style="width: 58%"></div></div>
	</td>
	<td data-value="42.11" class="pct low">42.11%</td>
	<td data-value="19" class="abs low">8/19</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="4" class="abs low">0/4</td>
	<td data-value="28.57" class="pct low">28.57%</td>
	<td data-value="7" class="abs low">2/7</td>
	<td data-value="42.11" class="pct low">42.11%</td>
	<td data-value="19" class="abs low">8/19</td>
	</tr>

<tr>
	<td class="file medium" data-value="Home.js"><a href="Home.js.html">Home.js</a></td>
	<td data-value="55.56" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 55%"></div><div class="cover-empty" style="width: 45%"></div></div>
	</td>
	<td data-value="55.56" class="pct medium">55.56%</td>
	<td data-value="27" class="abs medium">15/27</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="12" class="abs medium">6/12</td>
	<td data-value="55.56" class="pct medium">55.56%</td>
	<td data-value="27" class="abs medium">15/27</td>
	</tr>

<tr>
	<td class="file low" data-value="Login.js"><a href="Login.js.html">Login.js</a></td>
	<td data-value="40" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 40%"></div><div class="cover-empty" style="width: 60%"></div></div>
	</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="30" class="abs low">12/30</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="9" class="abs low">3/9</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="30" class="abs low">12/30</td>
	</tr>

<tr>
	<td class="file low" data-value="PasswordReset.js"><a href="PasswordReset.js.html">PasswordReset.js</a></td>
	<td data-value="26.09" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 26%"></div><div class="cover-empty" style="width: 74%"></div></div>
	</td>
	<td data-value="26.09" class="pct low">26.09%</td>
	<td data-value="23" class="abs low">6/23</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="6" class="abs low">0/6</td>
	<td data-value="14.29" class="pct low">14.29%</td>
	<td data-value="7" class="abs low">1/7</td>
	<td data-value="26.09" class="pct low">26.09%</td>
	<td data-value="23" class="abs low">6/23</td>
	</tr>

<tr>
	<td class="file medium" data-value="Progress.js"><a href="Progress.js.html">Progress.js</a></td>
	<td data-value="62.5" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 62%"></div><div class="cover-empty" style="width: 38%"></div></div>
	</td>
	<td data-value="62.5" class="pct medium">62.5%</td>
	<td data-value="16" class="abs medium">10/16</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="25" class="pct low">25%</td>
	<td data-value="8" class="abs low">2/8</td>
	<td data-value="62.5" class="pct medium">62.5%</td>
	<td data-value="16" class="abs medium">10/16</td>
	</tr>

<tr>
	<td class="file low" data-value="Register.js"><a href="Register.js.html">Register.js</a></td>
	<td data-value="29.63" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 29%"></div><div class="cover-empty" style="width: 71%"></div></div>
	</td>
	<td data-value="29.63" class="pct low">29.63%</td>
	<td data-value="27" class="abs low">8/27</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="10" class="abs low">0/10</td>
	<td data-value="22.22" class="pct low">22.22%</td>
	<td data-value="9" class="abs low">2/9</td>
	<td data-value="29.63" class="pct low">29.63%</td>
	<td data-value="27" class="abs low">8/27</td>
	</tr>

<tr>
	<td class="file low" data-value="Settings.js"><a href="Settings.js.html">Settings.js</a></td>
	<td data-value="46.15" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 46%"></div><div class="cover-empty" style="width: 54%"></div></div>
	</td>
	<td data-value="46.15" class="pct low">46.15%</td>
	<td data-value="13" class="abs low">6/13</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="0" class="abs high">0/0</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	<td data-value="46.15" class="pct low">46.15%</td>
	<td data-value="13" class="abs low">6/13</td>
	</tr>

<tr>
	<td class="file low" data-value="Workout.js"><a href="Workout.js.html">Workout.js</a></td>
	<td data-value="16" class="pic low">
	<div class="chart"><div class="cover-fill" style="width: 16%"></div><div class="cover-empty" style="width: 84%"></div></div>
	</td>
	<td data-value="16" class="pct low">16%</td>
	<td data-value="100" class="abs low">16/100</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="25" class="abs low">0/25</td>
	<td data-value="3.03" class="pct low">3.03%</td>
	<td data-value="33" class="abs low">1/33</td>
	<td data-value="16.84" class="pct low">16.84%</td>
	<td data-value="95" class="abs low">16/95</td>
	</tr>

<tr>
	<td class="file medium" data-value="initial.js"><a href="initial.js.html">initial.js</a></td>
	<td data-value="65.22" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 65%"></div><div class="cover-empty" style="width: 35%"></div></div>
	</td>
	<td data-value="65.22" class="pct medium">65.22%</td>
	<td data-value="23" class="abs medium">15/23</td>
	<td data-value="33.33" class="pct low">33.33%</td>
	<td data-value="6" class="abs low">2/6</td>
	<td data-value="50" class="pct medium">50%</td>
	<td data-value="10" class="abs medium">5/10</td>
	<td data-value="65.22" class="pct medium">65.22%</td>
	<td data-value="23" class="abs medium">15/23</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Fri Nov 13 2020 18:31:04 GMT-0600 (Central Standard Time)
            </div>
        </div>
        <script src="../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../sorter.js"></script>
        <script src="../block-navigation.js"></script>
    </body>
</html>
    