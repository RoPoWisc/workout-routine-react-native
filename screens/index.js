import React from 'react'

export const YourProgress = ({navigation}) => <YourProgress navigation={navigation} name="Your Progress" />
export const AddWorkout = ({navigation}) => <AddWorkout navigation={navigation} name="Add Workout" />
export const EditWorkout = ({navigation}) => <EditWorkout navigation={navigation} name="Edit Workout" />
export const Settings = ({navigation}) => <Settings navigation={navigation} name="Settings" />
//export const Login = ({navigation}) => <Login navigation={navigation} name="Login" />
