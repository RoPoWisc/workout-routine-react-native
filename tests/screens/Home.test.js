import React from 'react';
import renderer from 'react-test-renderer';
import mockStore from 'redux-mock-store';

import {Home} from '../../screens/Home';


describe('Home', () => {
    test('Page', () => {
        const tree = renderer.create(<Home />).toJSON();
        //console.warn(tree);
        expect(tree.children.length).toBe(2);
    });
    
    test('renders', () => {
        const render = renderer.create(<Home />).toJSON();
        expect(render).toMatchSnapshot();
    })
    let findTextElement = function(tree, element){
        console.warn(tree)
        return true;
    }
     
    it('Find text element', ()=>{
       let tree = renderer.create(
           <Home />
       ).toJSON();
     
       expect(findTextElement(tree, 'email')).toBeDefined();
    })
    it('check function', () => {
        let loghand = renderer.create(<Home />).getInstance();
        expect(loghand.componentDidMount()).toBeDefined();


        
    })
    it('check function', () => {
        let loghand = renderer.create(<Home />).getInstance();
        expect(loghand.onPressWorkoutButton()).toBeDefined();


        
    })

});

