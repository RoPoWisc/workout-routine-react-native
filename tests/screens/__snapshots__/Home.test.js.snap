// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders 1`] = `
<View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      appearance="default"
      level="1"
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
          },
          Object {
            "alignItems": "center",
            "flex": 1,
            "flexDirection": "row",
            "marginTop": 26.68,
          },
        ]
      }
    >
      <View
        appearance="default"
        level="1"
        style={
          Array [
            Object {
              "backgroundColor": "#FFFFFF",
            },
            Object {
              "alignItems": "flex-start",
              "flex": 3,
              "marginBottom": 26.68,
              "marginLeft": 15,
            },
          ]
        }
      >
        <Text
          appearance="default"
          category="s1"
          style={
            Array [
              Object {
                "color": "#222B45",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "600",
              },
              Object {
                "fontSize": 86.71000000000001,
                "fontWeight": "bold",
                "marginLeft": 15,
              },
            ]
          }
        >
          Personal
        </Text>
        <Text
          appearance="hint"
          category="p1"
          style={
            Array [
              Object {
                "color": "#8F9BB3",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
              },
              Object {
                "fontSize": 60.03,
                "marginLeft": 15,
                "textAlign": "center",
              },
            ]
          }
        >
          Workouts
        </Text>
      </View>
      <View
        appearance="default"
        level="1"
        style={
          Array [
            Object {
              "backgroundColor": "#FFFFFF",
            },
            Object {
              "alignSelf": "flex-start",
              "flex": 1,
              "flexDirection": "row",
              "justifyContent": "flex-end",
              "marginRight": 3.75,
            },
          ]
        }
      >
        <View
          accessible={true}
          focusable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "height": 93.38000000000001,
              "marginRight": 15,
              "marginTop": 16.008,
              "opacity": 1,
              "width": 105.00000000000001,
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "height": 93.38000000000001,
                "marginRight": 15,
                "marginTop": 16.008,
                "width": 105.00000000000001,
              }
            }
          />
        </View>
      </View>
    </View>
    <View
      appearance="default"
      level="1"
      style={
        Array [
          Object {
            "backgroundColor": "#FFFFFF",
          },
          Object {
            "alignItems": "center",
            "flex": 3,
            "justifyContent": "center",
            "marginHorizontal": 15,
          },
        ]
      }
    >
      <View
        appearance="default"
        level="1"
        style={
          Array [
            Object {
              "backgroundColor": "#FFFFFF",
            },
            Object {
              "flex": 1,
              "marginTop": -44.022000000000006,
            },
          ]
        }
      >
        <RCTScrollView
          data={
            Array [
              Object {
                "key": "Workout 1",
                "name": 1,
              },
              Object {
                "key": "Workout 2",
                "name": 1,
              },
              Object {
                "key": "Workout 3",
                "name": 1,
              },
            ]
          }
          disableVirtualization={false}
          getItem={[Function]}
          getItemCount={[Function]}
          horizontal={true}
          initialNumToRender={10}
          keyExtractor={[Function]}
          maxToRenderPerBatch={10}
          onContentSizeChange={[Function]}
          onEndReachedThreshold={2}
          onLayout={[Function]}
          onMomentumScrollEnd={[Function]}
          onScroll={[Function]}
          onScrollBeginDrag={[Function]}
          onScrollEndDrag={[Function]}
          removeClippedSubviews={false}
          renderItem={[Function]}
          scrollEventThrottle={50}
          stickyHeaderIndices={Array []}
          updateCellsBatchingPeriod={50}
          viewabilityConfigCallbackPairs={Array []}
          windowSize={21}
        >
          <View>
            <View
              onLayout={[Function]}
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                  },
                  null,
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "borderColor": "white",
                    "borderRadius": 40,
                    "borderWidth": 5,
                    "height": 466.9,
                    "justifyContent": "center",
                    "marginBottom": 20.009999999999998,
                    "opacity": 1,
                    "width": 397.5,
                  }
                }
              >
                <View
                  appearance="default"
                  level="1"
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#FFFFFF",
                      },
                      Object {
                        "alignItems": "center",
                        "borderColor": "white",
                        "borderRadius": 40,
                        "borderWidth": 5,
                        "height": 466.9,
                        "justifyContent": "center",
                        "marginBottom": 20.009999999999998,
                        "width": 397.5,
                      },
                    ]
                  }
                >
                  <View
                    accessibilityIgnoresInvertColors={true}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "stretch",
                        "borderRadius": 75,
                        "flex": 1,
                        "justifyContent": "flex-end",
                        "opacity": 0.9,
                        "overflow": "hidden",
                        "resizeMode": "cover",
                      }
                    }
                  >
                    <Image
                      source={1}
                      style={
                        Array [
                          Object {
                            "bottom": 0,
                            "left": 0,
                            "position": "absolute",
                            "right": 0,
                            "top": 0,
                          },
                          Object {
                            "height": undefined,
                            "width": undefined,
                          },
                          undefined,
                        ]
                      }
                    />
                    <Text
                      appearance="default"
                      category="p1"
                      style={
                        Array [
                          Object {
                            "color": "#222B45",
                            "fontFamily": "System",
                            "fontSize": 15,
                            "fontWeight": "400",
                          },
                          Object {
                            "color": "white",
                            "fontSize": 30,
                            "fontWeight": "bold",
                          },
                        ]
                      }
                    >
                      Workout 1
                    </Text>
                  </View>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                  },
                  null,
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "borderColor": "white",
                    "borderRadius": 40,
                    "borderWidth": 5,
                    "height": 466.9,
                    "justifyContent": "center",
                    "marginBottom": 20.009999999999998,
                    "opacity": 1,
                    "width": 397.5,
                  }
                }
              >
                <View
                  appearance="default"
                  level="1"
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#FFFFFF",
                      },
                      Object {
                        "alignItems": "center",
                        "borderColor": "white",
                        "borderRadius": 40,
                        "borderWidth": 5,
                        "height": 466.9,
                        "justifyContent": "center",
                        "marginBottom": 20.009999999999998,
                        "width": 397.5,
                      },
                    ]
                  }
                >
                  <View
                    accessibilityIgnoresInvertColors={true}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "stretch",
                        "borderRadius": 75,
                        "flex": 1,
                        "justifyContent": "flex-end",
                        "opacity": 0.9,
                        "overflow": "hidden",
                        "resizeMode": "cover",
                      }
                    }
                  >
                    <Image
                      source={1}
                      style={
                        Array [
                          Object {
                            "bottom": 0,
                            "left": 0,
                            "position": "absolute",
                            "right": 0,
                            "top": 0,
                          },
                          Object {
                            "height": undefined,
                            "width": undefined,
                          },
                          undefined,
                        ]
                      }
                    />
                    <Text
                      appearance="default"
                      category="p1"
                      style={
                        Array [
                          Object {
                            "color": "#222B45",
                            "fontFamily": "System",
                            "fontSize": 15,
                            "fontWeight": "400",
                          },
                          Object {
                            "color": "white",
                            "fontSize": 30,
                            "fontWeight": "bold",
                          },
                        ]
                      }
                    >
                      Workout 2
                    </Text>
                  </View>
                </View>
              </View>
            </View>
            <View
              onLayout={[Function]}
              style={
                Array [
                  Object {
                    "flexDirection": "row",
                  },
                  null,
                ]
              }
            >
              <View
                accessible={true}
                focusable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "borderColor": "white",
                    "borderRadius": 40,
                    "borderWidth": 5,
                    "height": 466.9,
                    "justifyContent": "center",
                    "marginBottom": 20.009999999999998,
                    "opacity": 1,
                    "width": 397.5,
                  }
                }
              >
                <View
                  appearance="default"
                  level="1"
                  style={
                    Array [
                      Object {
                        "backgroundColor": "#FFFFFF",
                      },
                      Object {
                        "alignItems": "center",
                        "borderColor": "white",
                        "borderRadius": 40,
                        "borderWidth": 5,
                        "height": 466.9,
                        "justifyContent": "center",
                        "marginBottom": 20.009999999999998,
                        "width": 397.5,
                      },
                    ]
                  }
                >
                  <View
                    accessibilityIgnoresInvertColors={true}
                    style={
                      Object {
                        "alignItems": "center",
                        "alignSelf": "stretch",
                        "borderRadius": 75,
                        "flex": 1,
                        "justifyContent": "flex-end",
                        "opacity": 0.9,
                        "overflow": "hidden",
                        "resizeMode": "cover",
                      }
                    }
                  >
                    <Image
                      source={1}
                      style={
                        Array [
                          Object {
                            "bottom": 0,
                            "left": 0,
                            "position": "absolute",
                            "right": 0,
                            "top": 0,
                          },
                          Object {
                            "height": undefined,
                            "width": undefined,
                          },
                          undefined,
                        ]
                      }
                    />
                    <Text
                      appearance="default"
                      category="p1"
                      style={
                        Array [
                          Object {
                            "color": "#222B45",
                            "fontFamily": "System",
                            "fontSize": 15,
                            "fontWeight": "400",
                          },
                          Object {
                            "color": "white",
                            "fontSize": 30,
                            "fontWeight": "bold",
                          },
                        ]
                      }
                    >
                      Workout 3
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </RCTScrollView>
      </View>
      <View
        appearance="default"
        level="1"
        style={
          Array [
            Object {
              "backgroundColor": "#FFFFFF",
            },
            Object {
              "alignItems": "center",
              "alignSelf": "stretch",
              "backgroundColor": "#013A73",
              "borderTopEndRadius": 30,
              "borderTopStartRadius": 30,
              "flex": 1,
              "marginHorizontal": -15,
              "marginTop": 133.4,
            },
          ]
        }
      >
        <Text
          appearance="default"
          category="p1"
          style={
            Array [
              Object {
                "color": "#222B45",
                "fontFamily": "System",
                "fontSize": 15,
                "fontWeight": "400",
              },
              Object {
                "color": "white",
                "fontSize": 30,
                "fontWeight": "bold",
              },
            ]
          }
        >
          Past Workouts
        </Text>
        <RCTScrollView
          data={
            Array [
              Object {
                "key": "
Legz4Dayz",
                "name": "Workout 1",
              },
              Object {
                "key": "
Chest Day",
                "name": "Workout 2",
              },
              Object {
                "key": "
Bulk",
                "name": "Workout 3",
              },
              Object {
                "key": "
Cut",
                "name": "Workout 4",
              },
              Object {
                "key": "
Superset",
                "name": "Workout 5",
              },
              Object {
                "key": "
HIIT",
                "name": "Workout 6",
              },
              Object {
                "key": "
Core 24/7",
                "name": "Workout 7",
              },
              Object {
                "key": "
Cardio",
                "name": "Workout 8",
              },
            ]
          }
          disableVirtualization={false}
          getItem={[Function]}
          getItemCount={[Function]}
          horizontal={false}
          initialNumToRender={10}
          keyExtractor={[Function]}
          maxToRenderPerBatch={10}
          onContentSizeChange={[Function]}
          onEndReachedThreshold={2}
          onLayout={[Function]}
          onMomentumScrollEnd={[Function]}
          onScroll={[Function]}
          onScrollBeginDrag={[Function]}
          onScrollEndDrag={[Function]}
          removeClippedSubviews={false}
          renderItem={[Function]}
          scrollEventThrottle={50}
          stickyHeaderIndices={Array []}
          updateCellsBatchingPeriod={50}
          viewabilityConfigCallbackPairs={Array []}
          windowSize={21}
        >
          <View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 1
                : 
                
Legz4Dayz
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 2
                : 
                
Chest Day
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 3
                : 
                
Bulk
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 4
                : 
                
Cut
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 5
                : 
                
Superset
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 6
                : 
                
HIIT
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 7
                : 
                
Core 24/7
              </Text>
            </View>
            <View
              onLayout={[Function]}
              style={null}
            >
              <Text
                appearance="default"
                category="p1"
                style={
                  Array [
                    Object {
                      "color": "#222B45",
                      "fontFamily": "System",
                      "fontSize": 15,
                      "fontWeight": "400",
                    },
                    Object {
                      "alignSelf": "center",
                      "backgroundColor": "white",
                      "borderColor": "black",
                      "borderRadius": 40,
                      "borderWidth": 2,
                      "fontSize": 24,
                      "paddingHorizontal": 135,
                      "paddingVertical": 40.019999999999996,
                    },
                  ]
                }
              >
                Workout 8
                : 
                
Cardio
              </Text>
            </View>
          </View>
        </RCTScrollView>
      </View>
    </View>
  </View>
</View>
`;
